src/main/java/org/bytedeco/javacpp/tools/Parser.java

```diff
@@ -2515,9 +2545,22 @@ boolean function(Context context, DeclarationList declList) throws ParserExcepti
                 }
             }
 
+            boolean needWrapper;
```

Comment:
Could we more meaningfully name that variable something like "needUpcast"?

Reply:
Ok. Or needUpcastWrapper. It's true if we need a wrapper to upcast `this` and/or at least one argument.

Reply:
Sure, if you really want to have the word "wrapper" in there.